DICOMwebBrowser/DICOMwebBrowser.py

```diff
@@ -573,19 +574,10 @@ def connectToServer(self):
     else:
       try:
         # Get all studies
-        studies = []
-        offset = 0
-
-        while True:
-            subset = self.DICOMwebClient.search_for_studies(offset=offset)
-            if len(subset) == 0:
-                break
-            if subset[0] in studies:
-                # got the same study twice, so probably this server does not respect offset,
-                # therefore we cannot do paging
-                break
-            studies.extend(subset)
-            offset += len(subset)
+        studies = self.DICOMwebClient.search_for_studies(
```

Comment:
I like that the proposed implementation is simpler, but I'm not sure if it is compatible with simple servers. We want this to be compatible with [dcm.js DICOMweb server](https://github.com/dcmjs-org/dicomweb-server) and the [built-in DICOMweb server in Slicer](https://slicer.readthedocs.io/en/latest/user_guide/modules/webserver.html#dicomweb-endpoints) (with URL `http://localhost:2016/dicom`).